tests/testthat/test_deprecated.R

```diff
@@ -0,0 +1,12 @@
+library(testthat)
+library(dials)
+
+context("test for deprecation warnings")
+
+test_that("deprecation warnings appear", {
```

Comment:
I think you can remove this test in favor of the one that would come from `test_parameters.R`

Reply:
Spoke with @DavisVaughan ; will leave.